content/en/universal_service_monitoring/setup.md

```diff
@@ -0,0 +1,831 @@
+
+```shell
+docker run --cgroupns host \
+--pid host \
+-e DD_API_KEY="<DATADOG_API_KEY>" \
+-e DD_SYSTEM_PROBE_SERVICE_MONITORING_ENABLED=true \
+-v /var/run/docker.sock:/var/run/docker.sock:ro \
+-v /proc/:/host/proc/:ro \
+-v /sys/fs/cgroup/:/host/sys/fs/cgroup:ro \
+-v /sys/kernel/debug:/sys/kernel/debug \
+-v /lib/modules:/lib/modules:ro \
+-v /usr/src:/usr/src:ro \
+-v /var/tmp/datadog-agent/system-probe/build:/var/tmp/datadog-agent/system-probe/build \
+-v /var/tmp/datadog-agent/system-probe/kernel-headers:/var/tmp/datadog-agent/system-probe/kernel-headers \
+-v /etc/apt:/host/etc/apt:ro \
+-v /etc/yum.repos.d:/host/etc/yum.repos.d:ro \
+-v /etc/zypp:/host/etc/zypp:ro \
+-v /etc/pki:/host/etc/pki:ro \
+-v /etc/yum/vars:/host/etc/yum/vars:ro \
+-v /etc/dnf/vars:/host/etc/dnf/vars:ro \
+-v /etc/rhsm:/host/etc/rhsm:ro \
+-e DD_SYSTEM_PROBE_SERVICE_MONITORING_ENABLED=true \
+-e HOST_ROOT=/host/root \
+--security-opt apparmor:unconfined \
+--cap-add=SYS_ADMIN \
+--cap-add=SYS_RESOURCE \
+--cap-add=SYS_PTRACE \
+--cap-add=NET_ADMIN \
+--cap-add=NET_BROADCAST \
+--cap-add=NET_RAW \
+--cap-add=IPC_LOCK \
+--cap-add=CHOWN \
+gcr.io/datadoghq/agent:latest
+```
+
+{{% /tab %}}
+{{% tab "Docker Compose" %}}
+
+Add the following to your `docker-compose.yml` file:
+
+```yaml
+services:
+  ...
+  datadog:
+    ...
+    environment:
+     - DD_SYSTEM_PROBE_SERVICE_MONITORING_ENABLED='true'
+    volumes:
+     - /var/run/docker.sock:/var/run/docker.sock:ro
+     - /proc/:/host/proc/:ro
+     - /sys/fs/cgroup/:/host/sys/fs/cgroup:ro
+     - /sys/kernel/debug:/sys/kernel/debug
+     - /lib/modules:/lib/modules
+     - /usr/src:/usr/src
+     - /var/tmp/datadog-agent/system-probe/build:/var/tmp/datadog-agent/system-probe/build
+     - /var/tmp/datadog-agent/system-probe/kernel-headers:/var/tmp/datadog-agent/system-probe/kernel-headers
+     - /etc/apt:/host/etc/apt
+     - /etc/yum.repos.d:/host/etc/yum.repos.d
+     - /etc/zypp:/host/etc/zypp
+     - /etc/pki:/host/etc/pki
+     - /etc/yum/vars:/host/etc/yum/vars
+     - /etc/dnf/vars:/host/etc/dnf/vars
+     - /etc/rhsm:/host/etc/rhsm
+    cap_add:
+     - SYS_ADMIN
+     - SYS_RESOURCE
+     - SYS_PTRACE
+     - NET_ADMIN
+     - NET_BROADCAST
+     - NET_RAW
+     - IPC_LOCK
+     - CHOWN
+    security_opt:
+     - apparmor:unconfined
+```
+
+For optional HTTPS support, also add:
+
+```yaml
+services:
+  ...
+  datadog:
+    ...
+    environment:
+     - HOST_ROOT: '/host/root'
+    volumes:
+     - /:/host/root:ro
+```
+
+{{% /tab %}}
+{{% tab "Docker Swarm" %}}
+
+As `Docker Swarm` does not yet support the changing of `security_opt`, the operating system
+must not have a running `apparmor` instance.
+
+If the operating system does not have a running `apparmor` instance, then use the same `docker-compose.yml` file from the `Docker-Compose` [section][1] beside the field `security_opt`.
+
+[1]: /universal_service_monitoring/?tab=dockercompose#enabling-universal-service-monitoring
```

Comment:
```suggestion
[1]: /universal_service_monitoring/setup/?tab=dockercompose#enabling-universal-service-monitoring
```
Not sure if you can just use `?tab=dockercompose#enabling-universal-service-monitoring`